packages/fxa-content-server/tests/functional/sign_in.js

```diff
@@ -152,6 +152,34 @@ registerSuite('signin', {
         .then(testElementExists(selectors.SETTINGS.HEADER));
     },
 
+    'signin as an existing user': function () {
+      return (
+        this.remote
+          .then(createUser(email, PASSWORD, { preVerified: true }))
+          .then(openPage(ENTER_EMAIL_URL, selectors.ENTER_EMAIL.HEADER))
+          .then(fillOutEmailFirstSignIn(email, PASSWORD))
+
+          // success is seeing the error message.
```

Comment:
I think this should say seeing the header element is success?

Reply:
Agreed!